test/e2e/dra/deploy.go

```diff
@@ -307,14 +324,35 @@ func (d *Driver) SetUp(nodes *Nodes, resources app.Resources) {
 	gomega.Expect(numNodes).To(gomega.Equal(int32(len(pods.Items))), "number of proxy pods")
 
 	// Run registar and plugin for each of the pods.
+	content := strings.ReplaceAll(pluginClusterRole, "dra-kubelet-plugin", "dra-kubelet-plugin-"+d.Name)
+	d.createFromYAML(ctx, []byte(content), "")
 	for _, pod := range pods.Items {
 		// Need a local variable, not the loop variable, for the anonymous
 		// callback functions below.
 		pod := pod
 		nodename := pod.Spec.NodeName
+
+		// Authenticate the plugin so that it only has minimal permissions.
+		driverUserInfo := (&serviceaccount.ServiceAccountInfo{
+			Name:      pod.Name,
+			Namespace: d.f.Namespace.Name,
+			NodeName:  nodename,
+		}).UserInfo()
+		driverClientConfig := d.f.ClientConfig()
+		driverClientConfig.Impersonate = rest.ImpersonationConfig{
+			UserName: driverUserInfo.GetName(),
+			Groups:   driverUserInfo.GetGroups(),
+			Extra:    driverUserInfo.GetExtra(),
+		}
+		driverClient, err := kubernetes.NewForConfig(driverClientConfig)
+		content := strings.ReplaceAll(pluginClusterRoleBinding, "dra-kubelet-plugin-on-node", driverUserInfo.GetName())
+		content = strings.ReplaceAll(content, "dra-kubelet-plugin", "dra-kubelet-plugin-"+d.Name)
```

Comment:
`dra-kubelet-plugin` is the name of the role binding:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: dra-kubelet-plugin
...
```

This can't work in a multi-node cluster because it creates multiple bindings under the same name.

I wonder why that didn't show up in the CI - let's try again:
/test pull-kubernetes-kind-dra

Reply:
Uhh, the pull-kubernetes-kind-dra with label filter no longer runs DRA tests...

That's because the E2E framework "helpfully" adds some default skip expression which excludes `[Feature:.*]'. I'll update that to be label filter aware.